Vagrantfile

```diff
@@ -8,4 +8,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = "redash/dev"
   config.vm.synced_folder "./", "/opt/redash/current"
   config.vm.network "forwarded_port", guest: 5000, host: 9001
+  config.vm.provision "shell" do |s|
+    s.inline = "/opt/redash/current/install_requirements.sh"
```

Comment:
Thing is that the dependencies we need to install in `circle.yml` are not the same as what we would install in Vagrant. In Vagrant we can install a larger set of dependencies to make it easier to develop and test wider range of scenarios, while for Circle I prefer to install as less as possible to make the build faster.

I really don't expect there will be many other requirements files added (if any), so this is a "risk" we can take.